core/src/main/java/org/infinispan/interceptors/impl/BaseRpcInterceptor.java

```diff
@@ -28,16 +32,20 @@
  *
  * @author <a href="mailto:manik@jboss.org">Manik Surtani (manik@jboss.org)</a>
  * @author Mircea.Markus@jboss.com
- * @deprecated Since 8.2, no longer public API.
+ * @since 9.0
  */
-@Deprecated
-public abstract class BaseRpcInterceptor extends CommandInterceptor {
+public abstract class BaseRpcInterceptor extends DDSequentialInterceptor {
+   private static final Log log = LogFactory.getLog(BaseRpcInterceptor.class);
```

Comment:
the log is not needed since it is an abstract class and you can transform the ```getLog()``` method to an abstract method.